CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java

```diff
@@ -97,6 +100,16 @@ public Replicator configure(Args args) throws Exception {
         if (heartbeat != null && !heartbeat.trim().isEmpty()){
             config.setHeartbeat(Integer.parseInt(heartbeat));
         }
+
+        if (maxRetries != null && !maxRetries.trim().isEmpty()) {
+            config.setMaxAttempts(Integer.parseInt(maxRetries));
+        }
+
+        if (maxTimeout != null && !maxTimeout.trim().isEmpty()) {
+            config.setMaxAttemptWaitTime(Integer.parseInt(maxTimeout));
```

Comment:
seems data type of setMaxAttemptWaitTime has been defined so differently on each platform? at least I see here java is int, dotnet is long then convert to timespan, and ios is double. are you sure it's expected or did you ask dev to give a reason?

Reply:
it will be changed when new api comes